Bug fixes (#6376)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

\* The official release date for this new format of release notes is May 15th, 2024. Historical release notes for prior dates may not reflect all available features released earlier this year or their tenancy availability.

## October 2024
 

> - **Fix:** Previously, POST requests to the Jobs API with invalid cron strings would return 500s but would update the underlying entity. Now, POST requests to the Jobs API with invalid cron strings will result in 400s, without the underlying entity being updated.

Reviewer: ```suggestion
- **Fix:** Previously, POST requests to the Jobs API with invalid `cron` strings would return HTTP response status code 500s but would update the underlying entity. Now, POST requests to the Jobs API with invalid `cron` strings will result in status code 400s, without the underlying entity being updated.
```

> - **Fix:** Corrected an issue where the `Source` view page in Explorer did not properly display source frehsness status if older than 30 days.

Author: ```suggestion
- **Fix:** Fixed an issue where the `Source` view page in dbt Explorer did not correctly display source freshness status if older than 30 days.
```

> - **Fix:** The UI now indicates when the description of a model is inhereted from a catalog comment.

Author: ```suggestion
- **Fix:** The UI now indicates when the description of a model is inherited from a catalog comment.
```
